---
layout: page
title: Speakers
permalink: /speakers/
---
<h3><a href="https://www.microsoft.com/en-us/research/people/ndalchau/">Neil Dalchau</a></h3>
  <p>
     <img src="http://qsbworkshop.github.io/qsb2018/images/dalchau.png" alt="Neil Dalchau" style="float:Left;width:140px;padding-right:10px;"><b>Talk: </b>To be announced<br><br><b>Bio: </b>I am a Scientist in the Biological Computation research group at Microsoft Research Cambridge. I am interested in how to program computation and decision-making in biological systems. The applications of programmed biology are numerous, including the synthesis of medicines and industrial chemicals, through to the direct treatment of disease. I studied Mathematics at the University of Oxford, UK (2001-2005), before becoming a biologist at the University of Cambridge. My PhD project was a collaboration between Alex Webbâ€˜s group at the Department of Plant Sciences and Jorge Goncalves in the Control Group at the Department of Engineering. Following my PhD, I briefly held a research associate position in the Control Group, working with Glenn Vinnicombe on applications of stochastic control theory to gene networks. I came to Microsoft Research as a postdoc in 2009, during which I worked with Andrew Phillips on modelling immune systems and synthetic gene networks. 
  </p>
<br>
<br>
<br>
<br>

<h3><a href="http://www.astbury.leeds.ac.uk/people/staff/staffpage.php?StaffID=SAH">Sarah Harris</a></h3>
  <p>
     <img src="http://qsbworkshop.github.io/qsb2018/images/HarrisS.jpg" alt="Sarah Harris" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b>Supercomputing in the Cellular Jungle<br><br><b>Bio: </b>The research groups uses high performance computing to model biomolecules and aim to cover all of the time and length-scales of relevance to molecular biology. This requires computational methods ranging from quantum mechanical, through atomistic to mesoscale modelling.
  </p>
<br>
<br>

<h3><a href="https://michellekendall.github.io/">Michelle Kendall</a></h3>
  <p>
     <img src="http://qsbworkshop.github.io/qsb2018/images/Michelle_Kendall.jpg" alt="Michelle Kendall" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b>Inferring the "true" tree: what to do when phylogenetic analyses disagree<br><br><b>Bio: </b>I am a senior researcher in statistical genetics and pathogen dynamics at the Oxford Big Data Institute working on the BEEHIVE project which is a cross-European study of HIV genomics and virulence. I am particularly interested in developing tree-based methods for answering statistical questions about HIV genotypic variation within and between patients.  </p>
<br>
<br>

<h3><a href="http://people.cryst.bbk.ac.uk/~ubcg71a/research.html">Irilenia Nobeli</a></h3>
  <p>
     <img src="http://qsbworkshop.github.io/qsb2018/images/nobeli.jpg" alt="Irilenia Nobeli" style="float:Left;width:140px;padding-right:10px;padding-bottom:90px;"><b>Talk: </b>To be announced<br><br><b>Bio: </b>Irilenia Nobeli (Birkbeck) focuses on the computational study of gene expression using transcriptomic data and is particularly excited about the roles of non-coding RNAs in regulation of gene expression. Her group has a long history of applying computational methods (mainly docking) to identify promising leads in drug design projects, or ligands relevant to protein function. In the past, we have also explored ligand-binding and catalytic promiscuity and its evolutionary origins. The group also has a long-standing interest in the analysis of the collections of endogenous metabolites in model organisms from a structural and physicochemical point of view. We use chemoinformatics methods to reveal the relationships between metabolites within a species, compare metabolites from different species, and finally compare endogenous metabolites to exogenous (human made or environmental) small molecules.
  </p>
<br>
<br>

<h3><a href="http://kityates.com/">Kit Yates</a></h3>
  <p>
     <img src="http://qsbworkshop.github.io/qsb2018/images/kityates.jpg" alt="Kit Yates" style="float:Left;width:140px;padding-right:10px;padding-bottom:50px;"><b>Talk: </b><em>Hybrid frameworks for modelling reaction-diffusion processes</em><br><br><b>Bio: </b>Kit Yates is a Senior Lecturer in the Department of Mathematical Sciences and co-director of the Centre for Mathematical Biology at the University of Bath. He completed his PhD in Mathematics at the University of Oxford in 2011. His research demonstrates that mathematics can be used to describe all sorts of real-world phenomena: from embryo formation to locust swarming and from sleeping sickness to egg-shell patterning. He is particularly interested in the role that randomness plays in Biology. Some specific areas of work include cell migration, spatial hybrid simulation methods for reaction-diffusion systems and methodologies in stochastic simulations.
  </p>
<br>
<br>

<h3><a href="https://www.birmingham.ac.uk/staff/profiles/cancer-genomic/yau-christopher.aspx">Christopher Yau</a></h3>
  <p>
     <img src="http://qsbworkshop.github.io/qsb2018/images/chrisyau.jpg" alt="Chris Yau" style="float:Left;width:140px;padding-right:10px;"><b>Talk: </b><em>Learning biological dynamics from static data: constructing temporal trajectories using machine learning</em><br><br><b>Bio: </b>Christopher Yau is a Reader in Computational Biology at the Institute of Cancer and Genomic Sciences where he is based at the Centre for Computational Biology and leads the Statistical Machine Learning BioHealth group. He is an expert in statistical methodologies for machine learning and data science and specialises in genomic science particularly cancer. His research ranges from mathematical and statistical algorithm development to collaborations with experimental scientists and clinicians involving modelling real world biomedical data sets. He leads a diverse group of researchers who specialise in both experimental and computational modelling and regularly gives talks and lectures around the world on data science. His goal is to conceive of a computational intelligence framework that will provide the foundation of learning health systems that will support novel health-related research from the molecular scale through to whole populations. 
  </p>
